cmd/kubeadm/app/phases/controlplane/manifests.go

```diff
@@ -255,6 +255,12 @@ func getAPIServerCommand(cfg *kubeadmapi.MasterConfiguration, k8sVersion *versio
 		} else {
 			command = append(command, fmt.Sprintf("--audit-log-maxage=%d", *cfg.AuditPolicyConfiguration.LogMaxAge))
 		}
+		if cfg.AuditPolicyConfiguration.WebhookConfigPath != nil {
+			command = append(command, "--audit-webhook-config-file="+kubeadmconstants.GetStaticPodAuditWebhookConfigFile())
+		}
+		if cfg.AuditPolicyConfiguration.WebhookInitialBackoff != nil {
+			command = append(command, fmt.Sprintf("--audit-webhook-initial-backoff=%d", *cfg.AuditPolicyConfiguration.WebhookInitialBackoff))
```

Comment:
Watch out for this nil pointer dereference error -- the LogMaxAge has a use as a 0 value so it has to be a pointer (which would become nil) to indicate if no value was set to get a default value that is not 0.

Reply:
Apiserver has a default built in value for --audit-webhook-initial-backoff, my intention is to omit the argument if not provided. I'll take a closer look at this today.